Doc/library/time.rst

```diff
@@ -388,6 +388,22 @@ Functions
       by a signal, except if the signal handler raises an exception (see
       :pep:`475` for the rationale).
 
+.. function:: sleep_until(secs)
+
+   Like :func:`sleep`, but sleep until the specified time of the system clock (as
+   returned by :func:`time`). This can be used, for example, to schedule events
+   at a specific timestamp obtained from
+   :meth:`datetime.timestamp <datetime.datetime.timestamp>`.
+
+   See the notes in :func:`sleep` on the behavior when interrupted and on accuracy.
+   Additional potential sources of inaccuracies include:
+
+   * Because this function uses the system clock as a reference, this means the
+     reference clock is adjustable and may jump backwards.
```

Comment:
It's unclear to me what is the *expected behavior* when the system clock jumps backward or forward. Does the function sleeps longer/shorter?

Reply:
If a caller asks to sleep until an absolute time on a given clock and that clock then "jumps", the caller should still wake up when that absolute time has elapsed. For example, if the clock is set backwards after the caller blocks, then the caller should sleep for a longer duration because it will take a longer time for the clock to reach the chosen absolute value. This is not a violation of the caller's request: they asked to block _until_ an absolute time, not to block _for_ a given duration.

If sleeping a longer/shorter duration is unacceptable for a caller, then either (a) the caller should use a relative sleep, or (b) the caller should schedule the sleep against a clock that cannot "jump".